include/Lcomplex.h

```diff
@@ -45,8 +45,6 @@
 
 #pragma GCC system_header
 
-#include <bits/c++config.h>
-
 //no longer include:
 //#include <bits/cpp_type_traits.h>  only thing used was is_floating... 
 //gcc 4.0 cpp_type_traits.h is not compatible with gcc 3.3. 
```

Comment:
In this file, our patches have another change `friend reset` to `friend void reset` around line ~135. Is it not useful anymore? I will inspect the code later.